hatchet_sdk/clients/admin.py

```diff
@@ -41,33 +41,33 @@ def new_admin(config: ClientConfig):
     return AdminClient(config)
 
 
-class ScheduleTriggerWorkflowOptions(TypedDict):
+class ScheduleTriggerWorkflowOptions(TypedDict, total=False):
     parent_id: Optional[str]
     parent_step_run_id: Optional[str]
     child_index: Optional[int]
     child_key: Optional[str]
     namespace: Optional[str]
 
 
-class ChildTriggerWorkflowOptions(TypedDict):
+class ChildTriggerWorkflowOptions(TypedDict, total=False):
     additional_metadata: Dict[str, str] | None = None
     sticky: bool | None = None
 
 
-class ChildWorkflowRunDict(TypedDict):
+class ChildWorkflowRunDict(TypedDict, total=False):
     workflow_name: str
     input: Any
     options: ChildTriggerWorkflowOptions
     key: str | None = None
 
 
-class TriggerWorkflowOptions(ScheduleTriggerWorkflowOptions, TypedDict):
```

Comment:
don't need this second base class since the parent already inherits from `TypedDict`